src/libs/antares/study/parts/thermal/cluster.cpp

```diff
@@ -479,6 +489,11 @@ void Data::ThermalCluster::reset()
     marketBidCost = 0.;
     variableomcost = 0.;
     costsTimeSeries.resize(1, CostsTimeSeries());
+    powerIncreaseCost = 0;
+    powerDecreaseCost = 0;
+
+    maxUpwardPowerRampingRate = 0;
+    maxDownwardPowerRampingRate = 0;
```

Comment:
```suggestion
    ramping.reset();
```

Reply:
Implemented in commit [b9801fe](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1817/commits/b9801feaf09d246c7f326859a24d4c2b97674d71)